fedimint-core/src/config.rs

```diff
@@ -120,6 +120,8 @@ pub struct PeerUrl {
 /// This includes global settings and client-side module configs.
 #[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize, Encodable, Decodable)]
 pub struct ClientConfig {
+    #[serde(default)]
```

Comment:
We should add a better comment about requirements and constrains of evolving `ClientConfig` as from the top of my head, I don't remember them.

I think now that `ClientConfig` it's not signed a new field should be fine, and old clients will just ignore it, so LGTM?